Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jakechampion/heroku-nodejs
```

Comment:
Any reason why we can't use the official https://hub.docker.com/r/heroku/nodejs/~/dockerfile/